src/index.js

```diff
@@ -0,0 +1,67 @@
+import './index.css';
+import getProducts from './modules/getProducts.js';
+import itemPopup from './modules/itemPopup.js';
+import closePopup from './modules/closePopUp.js';
+import getResFormData, { getCommentsFormData } from './modules/getResFormData.js';
+
+import commentPopup from './modules/commentPopup.js';
+
+import TheLikes from './modules/showLikes.js';
+
+import productCount from './modules/productCount.js';
+
+const store = document.querySelector('.counter');
+
+const logo = document.querySelector('.logo');
+logo.innerHTML = '<div class="oke">O K E</div>';
+
+// Items Interface
+const itemsContainer = async () => {
```

Comment:
- [ ] You might want to double check this again because it seems something is not right with this component as alleged by my colleague. It seems to be returning `0` and one other component. Here is an  attached video:
[Screencast from 2023-09-16 23-54-20.webm](https://github.com/ometman/JS-capstone-kanban/assets/99715640/92399919-918e-43bc-93d0-cd205c82340b)